src/client/browser/idle-page/status-markup-creator.js

```diff
@@ -0,0 +1,124 @@
+const SPINNER_CENTER_LEFT = 200;
+const SPINNER_CENTER_TOP  = 200;
+const SPINNER_RADIUS      = 196;
+
+const MESSAGE_BOX_TOP_INDENT  = 148;
```

Comment:
In UI messabox is a dialog window with message. But here it more likely a `status-text-area` or something like that.